Configuration/Settings.yaml

```diff
@@ -106,9 +106,18 @@ Flowpack:
 Neos:
   ContentRepository:
     Search:
+      # API. If set to FALSE, only index the "live" workspace and not user workspaces.
+      # If you only index the live workspace, Search will not work for your editors in the user workspaces.
+      # Furthermore, if you heavily rely on Search for collecting content, this might be strange for editors to
+      # work with -- as unpublished changes are not indexed right away.
       indexAllWorkspaces: true
+
       elasticSearch:
-        indexName: typo3cr
+
+        # API. name of the Elasticsearch index to use. Will create many indices prefixed by this indexName.
+        indexName: neoscr
```

Comment:
This one is breaking when you make your own mapping adjustments, like we did here: https://github.com/Flowpack/Flowpack.SearchPlugin/pull/26/files#diff-d06e40a2a4877cacd0ccde3ab14e8588R23
So we need some migration instructions for that.